services/server/youtube.ts

```diff
@@ -1,20 +1,24 @@
-type PlayListType = {
+export type YoutubeVideoLinkType = {
   title: string;
   link: string;
-}[];
+};
+
+const defaultVideoCount = 6;
 
 export async function getYoutubePlaylist(
   playlistId: string,
   videosCount: number
-): Promise<PlayListType> {
-  const url = `https://youtube.googleapis.com/youtube/v3/playlistItems?part=contentDetails&part=id&part=snippet&part=status&maxResults=100&playlistId=${playlistId}&key=${process.env.YOUTUBE_PRIVATE_KEY}`;
+): Promise<YoutubeVideoLinkType[]> {
+  const url = `https://youtube.googleapis.com/youtube/v3/playlistItems?part=contentDetails&part=id&part=snippet&part=status&maxResults=${
+    videosCount ?? defaultVideoCount
```

Comment:
```suggestion
    videosCount || defaultVideoCount
```
video count isn't nullable so it may not work as intended
![CleanShot 2024-01-03 at 17 02 52](https://github.com/SSWConsulting/SSW.Website/assets/600044/e44cb66b-4bb9-458f-9084-d5d3126202d8)